src/main/java/www/purple/mixxy/controllers/ApplicationController.java

```diff
@@ -1,49 +0,0 @@
-/**
- * Copyright (C) 2012 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package www.purple.mixxy.controllers;
-
-import ninja.FilterWith;
-import ninja.Result;
-import ninja.Results;
-import www.purple.mixxy.conf.ObjectifyProvider;
-import www.purple.mixxy.filters.UrlNormalizingFilter;
-
-import com.google.inject.Singleton;
-
-import ninja.appengine.AppEngineFilter;
-
-@Singleton
-@FilterWith({ AppEngineFilter.class })
```

Comment:
No, no, no.  I meant that you *shouldn't change* this file, not that you should delete it.  `@FilterWith({ AppEngineFilter.class, UrlNormalizingFilter.class })` was perfectly fine.